ocs_ci/helpers/cnv_helpers.py

```diff
@@ -0,0 +1,74 @@
+"""
+Helper functions specific for CNV
+"""
+import logging
+
+from ocs_ci.helpers.helpers import create_unique_resource_name, create_resource
+from ocs_ci.ocs import constants
+from ocs_ci.utility import templating
+
+logger = logging.getLogger(__name__)
+
+
+def create_vm(
```

Comment:
can we add in factory function?

Reply:
Yes, I have plans for it. will create a factory in the next PR